basiclti/basiclti-common/src/java/org/sakaiproject/basiclti/util/SakaiBLTIUtil.java

```diff
@@ -2637,6 +2637,7 @@ private static Object handleGradebook(String sourcedid, HttpServletRequest reque
 				}
 				retval = retMap;
 			} else if (isDelete) {
+				g.setAssignmentScoreString(siteId,gradebookColumn.getId(), user_id, null, "External Outcome");
 				g.deleteAssignmentScoreComment(siteId, gradebookColumn.getId(), user_id);
```

Comment:
I'm seeing similar cases on this file and they treat the second line this way:

`				if ( StringUtils.isBlank(comment) ) {
					gradingService.deleteAssignmentScoreComment(siteId, gradebookColumn.getId(), user_id);
				} else {
					gradingService.setAssignmentScoreComment(siteId, gradebookColumn.getId(), user_id, comment);
				}`

Wondering if we should add that for this case too?
and
Could we maybe move it to a function so it's not repeated 4 times?

Reply:
Yep, not sure myself... I thought it was similar to the case here where setAssignmentScoreString sends a null grade (clearing it up as I understand it)

https://github.com/sakaiproject/sakai/commit/fda47d215d2e87c489011f3a42637c95a2f40bc3